app/assets/stylesheets/app/_base.scss

```diff
@@ -26,6 +26,7 @@ $orange: #ff9e00;
 $gray: #dedede;
 $red: #f00;
 $green: #0f0;
+$blue: #0000ff;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected "#0000ff" to be "#00f"   color-hex-length